skimage/data/__init__.py

```diff
@@ -132,6 +132,37 @@ def checkerboard():
     return load("chessboard_GRAY.png")
 
 
+def cell():
+    """Cell floating in saline.
+
+    This is a quantitative phase image retrieved from a digital hologram using
+    the Python library ``qpformat``. The image shows a cell with high phase
+    value, above the background phase.
+
+    Because of a banding pattern artifact in the background, this image is a
```

Comment:
@jni is this the image I preprocessed? If so, maybe add the preprocessing code I applied to the original github image in some multiline string in the function itself.

Reply:
just as a reference to other developers.